commands/command.go

```diff
@@ -30,20 +30,21 @@ var Verbs = map[string]func(api.API) Command{
 	"paste":     NewPaste,
 	"pause":     NewPause,
 	"play":      NewPlay,
-	"prev":      NewPrevious,
 	"previous":  NewPrevious,
+	"prev":      NewPrevious,
```

Comment:
Oh, it was supposed to be.

Just tried again, and GNU sort is sorting them this way.

```
$ echo -e 'prev":      NewPrevious,\nprevious":  NewPrevious,' | sort
previous":  NewPrevious,
prev":      NewPrevious,
```